### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by mvader**
Just the last commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,42 @@
...
+The motivation behind this library is to use Go as a source of truth for your models instead of the other way around and then generating Go code from a `.proto` file, which does not generate idiomatic code.
+
+Proteus scans all the code in the selected packages and generates protobuf messages for every exported struct (and all the ones that are referenced in any other struct, even though they are not exported). Also, the types that semantically are used as enumerations in Go are transformed into proper protobuf enumerations.
+
+We want to build proteus in a very extensible way, so every step of the generation can be hackable via plugins and everyone can adapt proteus to their needs without actually having to integrate functionality that does not play well with the core library. We are releasing the plugin feature after Go 1.8 is released, which includes the `plugin` package of the standard library.
+
+### Install
+
+```
+go get github.com/src-d/proteus/cli
```
Comment (mcuadros) [inline README.md]: `go get -v github.com/src-d/proteus/...`

File: README.md [hunk 2]
```diff
@@ -4,16 +4,42 @@
...
+- Generation of services based on exported functions and methods.
+- Generation of gRPC service implementations.
+
+### Contribute
+
+If you are interested on contributing to **proteus**, open an [issue](https://github.com/src-d/proteus/issues) explaining which missing functionality you want to work in, and we will guide you through the implementation, and tell you beforehand if that is a functionality we might consider merging in the first place.
+
+### License
+
+MIT, see [LICENSE](https://github.com/src-d/proteus/blob/master/LICENSE)
```
Comment (mcuadros) [inline README.md]: Just link to (/LICENSE)

**Comment by mvader** [comment]
More docs added @Serabe @mcuadros

**Comment by mvader** [comment]
fixed @mcuadros

**Pull request merged**